projects/common/src/time/page-time-range-preference.service.ts

```diff
@@ -34,22 +34,18 @@ export class PageTimeRangePreferenceService {
     return combineLatest([
       this.pageTimeRangeStringDictionary$,
       this.featureStateResolver.getFeatureState(ApplicationFeature.PageTimeRange),
-      this.featureStateResolver.getFeatureFlagValue(ApplicationFeature.TenantDefaultTimeRangeMap)
+      this.featureStateResolver.getFeatureFlagValue<Dictionary<string>>(ApplicationFeature.FeatureDefaultTimeRangeMap)
     ]).pipe(
-      map(([pageTimeRangeStringDictionary, featureState, tenantDefaultTimeRangeMap]) => {
+      map(([pageTimeRangeStringDictionary, featureState, featureDefaultTimeRangeMap]) => {
         if (featureState === FeatureState.Enabled) {
```

Comment:
can do the same thing here and remove another level of nesting. If disable, return global default.

Reply:
Gonna leave this one, its easier to read imo. It quickly reads what is behind the feature flag, and what the behavior is when disabled. Oppose to reading each condition more carefully with all of them flat